curriculum/challenges/english/20-upcoming-python/learn-algorithm-design-by-building-the-shortest-path-algorithm/6559d95d5883a5822742b20b.md

```diff
@@ -0,0 +1,63 @@
+---
+id: 6559d95d5883a5822742b20b
+title: Step 41
+challengeType: 20
+dashedName: step-41
+---
+
+# --description--
+
+Add a final `print()` call outside the `for` loop to space the output.
```

Comment:
This `print()` is not making any difference in the output.

Reply:
I swear it was 😅 Maybe after the last refactoring it has lost its purpose. I'll remove the step.